monitoring/definitions/repo_updater.go

```diff
@@ -206,8 +206,8 @@ func RepoUpdater() *monitoring.Container {
 						{
 							Name:              "sched_error",
 							Description:       "repositories schedule error rate",
-							Query:             `max(rate(src_repoupdater_sched_error[1m]))`,
-							Critical:          monitoring.Alert().GreaterOrEqual(1, nil).For(time.Minute),
+							Query:             `max(rate(src_repoupdater_sched_error[10m]))`,
+							Critical:          monitoring.Alert().GreaterOrEqual(1, nil).For(10 * time.Minute),
```

Comment:
I think you can keep query aggregation at `1m` for more granular data while increasing `.For` to 10 minutes as you have done here - what this will do is say "for the last 10 one-minute windows, if all windows are over the threshold _then_ alert"

```suggestion
							Query:             `max(rate(src_repoupdater_sched_error[1m]))`,
							Critical:          monitoring.Alert().GreaterOrEqual(1, nil).For(10 * time.Minute),
```

If you change both to 10m, errors will be occurring for a _very_ long time before an alert can trigger